packages/web3-eth/src/types.ts

```diff
@@ -0,0 +1,194 @@
+import { AccessList, Log, TransactionHash, TransactionInfo, Uncles } from 'web3-common';
+import {
+	Address,
+	HexString,
+	Numbers,
+	ValidTypes,
+	ValidReturnTypes,
+	HexString32Bytes,
+	HexString256Bytes,
+	HexStringBytes,
+	Uint,
+} from 'web3-utils';
+
+export enum ChainNames {
+	MAINNET = 'mainnet',
+	GOERLI = 'goerli',
+	KOVAN = 'kovan',
+	RINKEBY = 'rinkeby',
+	ROPSTEN = 'ropsten',
+}
+
+export enum HardForks {
+	CHAIN_START = 'chainstart',
+	HOMESTEAD = 'homestead',
+	DAO = 'dao',
+	TANGERINE_WHISTLE = 'tangerineWhistle',
+	SPURIOUS_DRAGON = 'spuriousDragon',
+	BYZANTIUM = 'byzantium',
+	CONSTANTINOPLE = 'constantinople',
+	PETERSBURG = 'petersburg',
+	ISTANBUL = 'istanbul',
+	BERLIN = 'berlin',
+	LONDON = 'london',
+}
+
+export interface Transaction {
```

Comment:
shouldnt we use same transaction def https://github.com/ChainSafe/web3.js/blob/2ea05a8e9b251f618bdc54f0319f0c45896822af/packages/web3-common/src/eth_execution_api.ts#L78

Reply:
Created an [issue](https://github.com/ChainSafe/web3.js/issues/4632) for this, and added it to [Web3Eth TODO list](https://github.com/ChainSafe/web3.js/issues/4608)